src/NRedisStack/Json/IJsonCommands.cs

```diff
@@ -205,6 +205,17 @@ public interface IJsonCommands
     /// <remarks><seealso href="https://redis.io/commands/json.set"/></remarks>
     bool Set(RedisKey key, RedisValue path, RedisValue json, When when = When.Always);
 
+    /// <summary>
+    /// Set json file from the provided file Path.
+    /// </summary>
+    /// <param name="key">The key.</param>
+    /// <param name="path">The path to set within the key.</param>
+    /// <param name="filePath">The path of the file to set.</param>
+    /// <param name="when">When to set the value.</param>
+    /// <returns>The disposition of the command</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.set"/></remarks>
+    bool SetFile(RedisKey key, RedisValue path, string filePath, When when = When.Always);
```

Comment:
Like you said.. I guess SetFile is a bad name. How about SetFromFile?

Reply:
Agree :+1: